modules/system/classes/PluginManager.php

```diff
@@ -855,6 +855,9 @@ public function deletePlugin($id)
         if ($pluginPath = self::instance()->getPluginPath($id)) {
             File::deleteDirectory($pluginPath);
         }
+
+        // actually remove the plugin from our internal container
+        unset($this->plugins[ $this->normalizeIdentifier($id) ]);
```

Comment:
Would be better to normalize it at the start of the method to pass through to all the other calls

Reply:
I didn't want to modify the current behavior, it's tricky code...

Reply:
Let's see what the unit tests say...

Reply:
Went well... there are no tests for deletePlugin()! :rofl: 

I'll try to add some tests/

Reply:
@LukeTowers I'm having trouble writing unit tests for deletePlugin() as it introduces calls to the UpdateManager class and I'm getting errors like the following:
```
1) PluginManagerTest::testDeletePlugin 
Illuminate\Database\QueryException: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO) (SQL: select * from `system_plugin_history` where `code` = October.Sample order by `id` asc) 
```

Do you mind if I revert to normalizing the code only for the unset() call to make sure that does not introduce any bugs, and we can revisit using the normalized code for the other calls in a different PR?

Reply:
Sure, that's fine